### PR TITLE
Adding Evaluate button for alerts to test them

### PR DESCRIPTION
**Pull request by ezraodio1**
## What type of PR is this? 

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] New Query Runner (Data Source) 
- [ ] New Alert Destination
- [ ] Other

## Description
Adding new button to evaluate an alert and test its condition. This button allows users to verify that the alert they created works how they intended it to. This PR is related to https://github.com/getredash/redash/discussions/7024.

## How is this tested?

- [x] Unit tests (pytest, jest)
- [ ] E2E Tests (Cypress)
- [ ] Manually
- [ ] N/A

## Related Tickets & Documents

(https://github.com/getredash/redash/discussions/7024)
## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

https://github.com/getredash/redash/assets/97199317/d1677b80-c249-4e11-aaa3-f19f61715bc4

### CODE REVIEW

File: client/app/pages/alert/Alert.jsx [hunk 1]
```diff
@@ -177,6 +178,17 @@ class Alert extends React.Component {
...
   };
 
+  evaluate = () => {
+    const { alert } = this.state;
+    return AlertService.evaluate(alert)
+    .then(() => {
+      notification.success("Alert Evaluated. Please refresh the page.");
+    })
+    .catch(() => {
+      notifications.error("Failed evaluating alert.")
```
Comment (eradman) [inline client/app/pages/alert/Alert.jsx:188]: Change to "Failed to evaluate alert."

File: client/app/pages/alert/Alert.jsx [hunk 2]
```diff
@@ -177,6 +178,17 @@ class Alert extends React.Component {
       });
   };
 
+  evaluate = () => {
+    const { alert } = this.state;
+    return AlertService.evaluate(alert)
+    .then(() => {
+      notification.success("Alert Evaluated. Please refresh the page.");
```
Comment (eradman) [inline client/app/pages/alert/Alert.jsx:185]: Suggested wording: "Alert evaluated. Refresh page for updated status."

**Review by eradman — CHANGES_REQUESTED** [review]

File: client/app/pages/alert/Alert.jsx (hunk 2 above)
Comment (ezraodio1) [reply client/app/pages/alert/Alert.jsx]: I made these changes

**Review by eradman — APPROVED** [review]
Super useful. Works as expected

**Comment by eradman** [comment]
@ezraodio1 after rebasing it appears that there is one restyled change that needs to be applied

**Pull request merged**